### PR TITLE
New post candidate, Imperfection allowed

### PR DESCRIPTION
**Pull request by eveliinalindgren**
A personal tale of reviving my dev skills at Solita after stalling professionally elsewhere. All comments appreciated!

### CODE REVIEW

File: _posts/2022-08-17-imperfection-allowed.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - impostor syndrome
+  - personal growth
+  - psychological safety
+---
+
+Ever feel like everyone except you manages to navigate the choppy waters of building and maintaining a career in professional software development? These high achievers sail from strength to strength, propelled along by a clear map, a well-calibrated compass and ample reserves of knowledge, skills and self-confidence. Whereas you're stuck on a rocky shore, unsure of where exactly you are and where you should be going. Scared that you don't have what it takes to reach the high seas.
+
+![Rocky outlook](/img/imperfection-allowed/rocky-outlook.JPG)
+
+Well, this was me a while back. I'd worked as a software developer for well over a decade, and to begin with things had progressed nicely. I joined a respectable company with interesting, societally impactful projects. I learnt various technologies (any MOSS 2007 survivors out there?), continued studying alongside work and enjoyed the many wonderful aspects of the dev profession.
```
Comment (hjhamala) [inline _posts/2022-08-17-imperfection-allowed.md:20]: Rephrase "and to begin with things had progressed nicely"

Comment (hjhamala) [inline _posts/2022-08-17-imperfection-allowed.md:20]: Perhaps open up MOSS abbreviation?

File: _posts/2022-08-17-imperfection-allowed.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Downward spirals
+
+However, gradually my outlook became rockier. I switched jobs. I didn't dare to apply to the best-of-class companies or the more challenging positions, because I've always had a tendency towards [impostor syndrome](https://dev.solita.fi/2021/02/12/generalist-specialization-and-the-impostor-syndrome.html). I had not built strong professional networks and whatever networks I had, I didn't dare utilize. Any kind of self-promotion felt extremely alien.
+
+Years went by and I ended up stalling professionally, which in this industry means going backwards. I was entangled in legacy systems, doing odd jobs and patchy maintenance. Working in a tiny dev team, which meant limited possibilities for collegial learning and support. Hello, [boreout](https://dev.solita.fi/2022/02/21/is-it-better-to-burn-out-than-fade-away.html)! I was deeply anxious I'd missed the up-to-date developer boat so completely I'd never climb back aboard.
```
Comment (hjhamala) [inline _posts/2022-08-17-imperfection-allowed.md:26]: Remove ", which" in Working in a tiny dev team ...

Comment (hjhamala) [inline _posts/2022-08-17-imperfection-allowed.md:26]: so completely "that"

File: _posts/2022-08-17-imperfection-allowed.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+![Entangled](/img/imperfection-allowed/tangle.JPG)
+
+## Breaking free from the tangle
+
+But behold! Last year I saw the light: things had to change. I applied to a few dev positions, again with zero skills in network utilization, social media presence, hobby project showcasing or any of the supposed must-dos in today's recruitment market. In a fit of bravery, I also sent an application to Solita, because I'd long had a positive image of the company. Moreover, I'd seen some Solita consultants in action and been wowed by their easy-going professionalism.
+
+Despite my disbelief, I ended up at Solita. I was honest right from the start concerning my imperfections, most notably the need for catching up regarding modern development. But it was all okay! Together we would find ways to get me going again in an environment that consciously invests in [psychological safety](https://www.solita.fi/en/blogs/feeling-safe-at-work-is-good-for-people-and-the-business/).
```
Comment (hjhamala) [inline _posts/2022-08-17-imperfection-allowed.md:34]: catching up regarding -> catching up with

**Review by hjhamala — APPROVED** [review]
In general this is quite good. Perhaps you could tell more openly about what legacy techs you did in your earlier life? What cool new things or change did Solita bring to you?

**Review by jarnovayrynen — APPROVED** [review]

**Pull request merged**